src/vmm/src/io_uring/operation/mod.rs

```diff
@@ -119,7 +119,7 @@ impl<T: Debug> Operation<T> {
 
     /// Consumes the operation and returns the associated `user_data`.
     pub fn user_data(self) -> T {
-        *self.user_data
+        self.user_data
```

Comment:
I think this getter does not need to be here. Just make `user_data` a public field.